packs/alertmanager/outputs.tpl

```diff
@@ -0,0 +1 @@
+alertmanager successfully deployed.
```

Comment:
Depending on what we decide about the Prometheus pack updates as well as routing, the output could include this information.

Reply:
happy to improve, but this seems to be the "default" response most packs provide at the moment. What would like to see added?

Reply:
updated somewhat